myia/api.py

```diff
@@ -72,44 +56,250 @@ def _convert_function(env, fn):
     return g
 
 
-def lax_converter():
-    """Return a "lax" converter that allows arbitrary objects."""
-    return Converter(
-        default_object_map(),
-        TypeMap({
-            FunctionType: _convert_function,
-            tuple: _convert_sequence,
-            list: _convert_sequence,
-            object: _convert_identity,
-            type: _convert_identity,
-        })
-    )
-
-
-default_vm = VM(vm_implementations,
-                py_implementations,
-                lax_converter())
+lax_type_map = TypeMap({
+    FunctionType: _convert_function,
+    tuple: _convert_sequence,
+    list: _convert_sequence,
+    object: _convert_identity,
+    type: _convert_identity,
+})
 
 
 def parse(func: FunctionType, resolve_globals=True) -> Graph:
     """Parse a function into ANF."""
-    converter = lax_converter()
-    resolver_opt = PatternEquilibriumOptimizer(
-        optlib.make_resolver(converter)
-    )
-    g = converter(func)
-    if resolve_globals:
-        resolver_opt(g)
-        g = clone(g)
+    pdef = standard_pipeline.select('parse', 'resolve')
+    pdef = pdef.configure(resolve=resolve_globals)
+    g = pdef.make()(input=func)['graph']
     return g
 
 
 def run(g: Graph, args: List[Any]) -> Any:
     """Evaluate a graph on a set of arguments."""
-    return default_vm.evaluate(g, args)
+    pdef = standard_pipeline.select('export')
+    f = pdef.make()(graph=g)['output']
+    return f(*args)
 
 
 def compile(obj):
     """Return a version of the function that runs using Myia's VM."""
-    myia_obj = default_vm.convert(obj)
-    return default_vm.export(myia_obj)
+    pdef = standard_pipeline.select(
+        'parse', 'resolve', 'export'
+    )
+    return pdef.make()(input=obj)['output']
+
+
+############
+# Pipeline #
+############
+
+
+class Converter(PipelineStep):
```

Comment:
This is apparently no longer a PipelineStep

Reply:
Indeed it isn't.